beacon_node/beacon_chain/src/canonical_head.rs

```diff
@@ -15,7 +15,7 @@
 //! 3. `Mutex<()>`: Is used to prevent concurrent execution of `BeaconChain::recompute_head`.
 //!
 //! This module has to take great efforts to avoid causing a deadlock with these three methods. Any
-//! developers working in this module should tread carefully and seek a detailed review.
+//! developers working in this module should treat carefully and seek a detailed review.
```

Comment:
```suggestion
//! developers working in this module should tread carefully and seek a detailed review.
```

False positive. "Tread" is the intended word here and makes more sense than "treat".